cli/trampolines/trampolines_aarch64.S

```diff
@@ -13,8 +13,9 @@ CNAME(name)##: SEP \
     br x16 SEP \
 .cfi_endproc SEP \
 
-JL_EXPORTED_FUNCS(XX)
+JL_RUNTIME_EXPORTED_FUNCS(XX)
 #ifdef _OS_WINDOWS_
-JL_EXPORTED_FUNCS_WIN(XX)
+JL_RUNTIME_EXPORTED_FUNCS_WIN(XX)
 #endif
+JL_CODEGEN_EXPORTED_FUNCS(XX)
```

Comment:
It feels a little odd to be exporting the internal plugins from here, but otherwise, this PR seems to be mostly just renaming lots of (lots of) variables, so seems very straightforward. Not sure why it would be failing on CI. Are we accidentally linking part of libLLVM statically?

Reply:
Yeah that's true; I only really did this to reuse the mechanism.